src/test/java/com/fittracker/fittracker/security/JwtUtilsTest.java

```diff
@@ -0,0 +1,104 @@
+package com.fittracker.fittracker.security;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.core.Authentication;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.lenient;
+
+@ExtendWith(MockitoExtension.class)
+class JwtUtilsTest {
+
+    private static final String TEST_SECRET = "3Wlamn5wuIm2698NNKnplSq2GY8vqbzBCRynuvh1swOKzp5Ivr1zaQ9seoEyk5mFNEyvkxvU9pExIqdm";
+    private static final int JWT_SIGNATURE_LENGTH = 86;
+    private static final int TEST_TOKEN_EXPIRATION_PERIOD_MINUTES = 60;
+    @Mock
+    private Authentication authentication;
+    private JwtUtils jwtUtils;
+
+    private JwtConfig jwtConfig;
+
+    @BeforeEach
+    public void setUp() {
+        lenient().when(authentication.getName()).thenReturn("user");
+        jwtConfig = new JwtConfig(TEST_SECRET, TEST_TOKEN_EXPIRATION_PERIOD_MINUTES);
+        jwtUtils = new JwtUtils(jwtConfig);
+    }
+
+
+    @Test
+    void givenNameInAuthentication_shouldReturnTokenInValidFormat() {
```

Comment:
If we don't use nested classes (like here, where we only have one test per method), we could include the tested method in the test name, like `generateToken_givenNameInAuthentication_shouldReturnTokenInValidFormat`